input/pagecontent/reference.md

```diff
@@ -1544,25 +1651,103 @@ A defined rule set can be applied to an item by using an [`insert` rule](#rule-s
-  Expression: "family.exists() or given.exists()"
-  Severity:   #error
-  XPath:      "f:given or f:family"
+  ValueSet: BodyWeightPreconditionVS
+  Title: "Body weight preconditions."
+  Description:  "Circumstances for body weight measurement."
+  * SCT#971000205103 "Wearing street clothes with shoes"
+  * SCT#961000205106 "Wearing street clothes, no shoes"
+  * SCT#951000205108 "Wearing underwear or less"
+  ```
+
+* Define a value set using [intensional](https://blog.healthlanguage.com/the-difference-between-intensional-and-extensional-value-sets) rules:
+
+  ```
+  ValueSet: HistologyMorphologyBehaviorVS
+  Id: mcode-histology-morphology-behavior-vs
+  Title: "Histology Morphology Behavior Value Set"
+  Description: "Codes representing the structure, arrangement, and behavioral characteristics of malignant neoplasms, and cancer cells.
+  * include codes from system SCT where concept is-a #367651003 "Malignant neoplasm of primary, secondary, or uncertain origin (morphologic abnormality)"
+  * include codes from system SCT where concept is-a #399919001 "Carcinoma in situ - category (morphologic abnormality)"
+  * include codes from system SCT where concept is-a #399983006 "In situ adenomatous neoplasm - category (morphologic abnormality)"
+  * exclude codes from system SCT where concept is-a #450893003 "Papillary neoplasm, pancreatobiliary-type, with high grade intraepithelial neoplasia (morphologic abnormality)"
+  * exclude codes from system SCT where concept is-a #128640002 "Glandular intraepithelial neoplasia, grade III (morphologic abnormality)"
+  * exclude codes from system SCT where concept is-a #450890000 "Glandular intraepithelial neoplasia, low grade (morphologic abnormality)"
+  * exclude codes from system SCT where concept is-a #703548001 "Endometrioid intraepithelial neoplasia (morphologic abnormality)"
   ```
 
-### Formal Grammar
+> **Note:** Intensional and extensional forms can be used together in a single value set definition.
+
+### Appendix: Abbreviations
+
+| Abbreviation | Description |
+|-----|----|
+| ANTLR4  | ANother Tool for Language Recognition, version 4
+| CQL | Clinical Quality Language
+| D  | Flag denoting draft status
+| FHIR  | Fast Healthcare Interoperability Resources
+| FSH   | FHIR Shorthand
+| IG | Implementation Guide
+| JSON   | JavaScript Object Notation
+| LNC | Common FSH alias for LOINC
+| LOINC | Logical Observation Identifiers Names and Codes
+| NPI   | National Provider Identifier (US)
+| N   |  Flag denoting normative element
+| MS  | Flag denoting a Must Support element
+|  OID   | Object Identifier
+| SCT | Common FSH alias for SNOMED Clinical Terms
+|  SD   | StructureDefinition
+|  SU  | Flag denoting "include in summary"
+|  TU  | Flag denoting trial use element
+| UCUM  | Unified Code for Units of Measure
+| URL    | Uniform Resource Locator
+| XML   | Extensible Markup Language
+
+### Appendix: Formal Grammar
```

Comment:
Don't forget to update this once we've completed all grammar changes.

Reply:
Leaving unresolved as a reminder.

Reply:
We can push this forward. I updated as of today, but I don't want to hold up this PR for the final 1.0 grammar (especially if we make array indexing changes)

Reply:
I agree.  Didn't mean to imply this should hold up the PR.